src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -148,11 +148,17 @@ public String getFeedbackSessionName() {
     }
 
     public String getStartTimeString() {
+        if (startTime == null) {
+            return "";
+        }
         Date startTimeInUtc = TimeHelper.convertLocalDateToUtc(startTime, timeZone);
         return TimeHelper.formatDateTimeForSessions(startTimeInUtc, timeZone);
     }
 
     public String getEndTimeString() {
+        if (endTime == null) {
+            return "";
```

Comment:
Maybe return `"-"` instead?
![image](https://user-images.githubusercontent.com/6365864/28512238-9f292576-7083-11e7-9f69-3c6399182645.png)
 Leaving a blank space there seems odd to me.